website/conferences/signals.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+
+import blinker
+
+signals = blinker.Namespace()
+
+osf4m_new_user = signals.signal('osf4m-new-user')
```

Comment:
As suggested in a previous comment, use `os4m_user_created` to follow current naming conventions.